balances/balances.go

```diff
@@ -0,0 +1,34 @@
+package balances
+
+import "github.com/checkout/checkout-sdk-go/common"
+
+const (
+	balances = "balances"
+)
+
+type (
+	Query struct {
+		Query string `url:"query,omitempty"`
```

Comment:
What is the difference of using `url`, I mean what is the difference during serialization?
Also, `Query`  name seems pretty simple because Query means anything, I said that we should add something specific like BalancesQuery or so

Reply:
The `url` tag is used by `google/go-querystring` package to specify the name of the fields when they are parsed as query values.

Reply:
Regarding the name of the struct, when its imported to another package it would read like `balances.Query` and I think that is specific enough, but if you want I can rename it to something like `balances.BalancesQuery` or `balances.QueryFilter`. Let me know what you think?